hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -998,24 +998,6 @@ possible configurations would overwhelm and obscure the important.
     See http://hbase.apache.org/book.html#offheap.blockcache for more information.
     </description>
   </property>
-  <property>
-    <name>hbase.hstore.compaction.throughput.lower.bound</name>
-    <value>52428800</value>
-    <description>The target lower bound on aggregate compaction throughput, in bytes/sec. Allows
-    you to tune the minimum available compaction throughput when the
-    PressureAwareCompactionThroughputController throughput controller is active. (It is active by
-    default.)</description>
-  </property>
-  <property>
-    <name>hbase.hstore.compaction.throughput.higher.bound</name>
-    <value>104857600</value>
-    <description>The target upper bound on aggregate compaction throughput, in bytes/sec. Allows
-    you to control aggregate compaction throughput demand when the
-    PressureAwareCompactionThroughputController throughput controller is active. (It is active by
-    default.) The maximum throughput will be tuned between the lower and upper bounds when
-    compaction pressure is within the range [0.0, 1.0]. If compaction pressure is 1.0 or greater
-    the higher bound will be ignored until pressure returns to the normal range.</description>
-  </property>
```

Comment:
What is the intention behind removal of these configs from hbase-default?

Reply:
I understand code has default value but let's keep this as is. It is easy to refer default values here.

Reply:
I found that :when I run a junit test  ,If the parameters is already in 'hbase-default.xml' ,after I change the value in 'hbase-site.xml' and I want to  use 'admin.updateConfiguration' to update it , the value is always the default value.

But I deploy a cluster just now ,It's not look like this. The parameters were updated in a cluster mode.

Reply:
> I understand code has default value but let's keep this as is. It is easy to refer default values here.

I found that :when I run a junit test ,If the parameters is already in 'hbase-default.xml' ,after I change the value in 'hbase-site.xml' and I want to use 'admin.updateConfiguration' to update it , the value is always the default value.

But I deploy a cluster just now ,It's not look like this. The parameters were updated in a cluster mode.

Reply:
> I understand code has default value but let's keep this as is. It is easy to refer default values here.

I found that :when I run a junit test ,If the parameters is already in 'hbase-default.xml' ,after I change the value in 'hbase-site.xml' and I want to use 'admin.updateConfiguration' to update it , the value is always the default value.

But I deploy a cluster just now ,It's not looks like this. The parameters were updated in a cluster mode.

Reply:
Maybe I should create another issue about how to update config in junit tests.

In this patch I can use onConfigurationChange to update it ,likes TestCompactionWithThroughputController